File: gang/week6/같은 숫자는 싫어.java
import java.util.*;

public class Solution {
    public int[] solution(int[] arr) {
        ArrayList<Integer> result = new ArrayList<>();
        for(int i = 0; i< arr.length - 1; i++) {
            if(arr[i] != arr[i + 1]) {  // 현재 값과 다음 값이 다르면
                result.add(arr[i]);     // 값을 배열엔 넣음
            }
        }

        result.add(arr[arr.length - 1]);    // arr 배열의 마지막 값을 넣음

        int[] answer = new int[result.size()];
        for(int i =0; i<answer.length; i++) {
            answer[i] = result.get(i);
        }
        return answer;
    }
}

File: yeongseong/6week/같은 숫자는 싫어.js
function solution(arr)
{
    var answer = [];
    
    for (let i = 0; i < arr.length; i++){
        if (arr[i] !== arr[i+1]){ // 현재 수와 다음 수가 다를 때
            answer.push(arr[i]);
        }
    }
    return answer;
}